src/openzaak/components/documenten/openapi.yaml

```diff
@@ -3925,12 +3318,9 @@ components:
           $ref: '#/components/schemas/Integriteit'
         informatieobjecttype:
           title: Informatieobjecttype
-          description: URL-referentie naar het INFORMATIEOBJECTTYPE (in de Catalogi
-            API).
+          description: Referentie naar het INFORMATIEOBJECTTYPE (in de Catalogi API).
           type: string
           format: uri
-          maxLength: 200
-          minLength: 1
```

Comment:
Should be fixed with general fix mentioned earlier.

Reply:
please take a look at https://github.com/open-zaak/open-zaak/pull/51#discussion_r315630538